app/assets/javascripts/users.js

```diff
@@ -0,0 +1,25 @@
+function getRepositories() {
+  var url = $("#repositories").data("repositories")
+  $.getJSON(url, function(data) {
+    $.each(data, function(_idx, repository) {
+      $("#repositories").append("<tr>")
+      $("#repositories").append("<td>" + repository.name + "</td>")
+      $("#repositories").append("<td><a href=\"" + repository.html_url + "\">Go</a></td>")
+      $("#repositories").append("<td>" + repository.stargazers_count + "</td>")
+      $("#repositories").append("<td>" + repository.forks + "</td>")
+      $("#repositories").append("</tr>")
+    })
+  })
+}
+
+function getTemperature() {
+  var url = $("#temperature").data("temp")
+  $.getJSON(url, function(data) {
+    $("#temperature").append(data.temperature)
+  })
+}
+
+$(document).ready(function() {
```

Comment:
Are you sure you want to add the request when the document is ready? a cooler way is to call it when you are rendering the repos or the temperature